docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -504,71 +504,18 @@ Status Codes:
 
 Start the container `id`
 
+> **Note**:
```

Comment:
For backwards compatibility, this endpoint accepts a `HostConfig` as a JSON-encoded request body.